test/addons/ocm-cluster/start

```diff
@@ -97,9 +98,21 @@ def join_cluster(cluster, hub):
 
 def accept_cluster(cluster, hub):
     print("Accepting cluster")
+
+    managed_clusters = clusteradm.get("clusters", output="yaml", context=hub)
+
+    managed_clusters = managed_clusters.split("---")
+    for doc in managed_clusters:
+
+        managed_cluster = yaml.safe_load(doc)
+        if managed_cluster["metadata"]["name"] == cluster and managed_cluster["spec"]["hubAcceptsClient"] == True:
```

Comment:
maybe rewrite this as `if managed_cluster["metadata"]["name"] == cluster and managed_cluster["spec"]["hubAcceptsClient"]` remove `True`. I think this is what the linter error is pointing to.